lib/vue/src/components/map/CMap.vue

```diff
@@ -307,6 +303,12 @@ export default {
 </script>
 
 <style scoped>
+.c-map-container {
+  height: 75vh;
```

Comment:
Not sure about this, why do we have to have a height defined in the component.
Since the view using it is what should define the height.
The map should fill the space that the parent component provides.